docker/develop/scripts/funcs.sh

```diff
@@ -201,9 +201,12 @@ function define_common_envvars() {
   set_env "GOOGLE_CLOUD_PROJECT_ID" "${APPLICATION_ID}"
   set_env "PROJECT_PREFIX" "${APPLICATION_ID//-/_}"
 
+  # Parallelism env var
+  set_env "CIRCLE_NODE_INDEX" "${CIRCLE_NODE_INDEX}" 0
```

Comment:
One thing to consider here is perhaps updating this statement to be this:

`set_env "CIRCLE_NODE_INDEX" "local"`

This would effectively:

1. Ensure that the value is always set to _something_
2. If already defined, leave alone
2. If not already defined, set the value to `"local"`

This could be used to easily determine if a given set of datasets / buckets was created from a ci pipeline run or from being run locally.

Reply:
Thanks, this will definitely be a useful feature.